products/PurePhone/services/db/databases/migration/settings_v2/0/up.sql

```diff
@@ -89,7 +89,11 @@ INSERT OR IGNORE INTO settings_tab (path, value) VALUES
     ('\ServiceTime\\gs_automatic_date_and_time_is_on', '1'),
     ('\ServiceEink\\display_inverted_mode', '0'),
     ('display_lock_screen_deep_refresh_rate', '30'),
-    ('cl_volte_enabled', '0');
+    ('cl_volte_enabled', '0'),
+    ('\ApplicationMeditation\\meditation_duration', '15'),
+    ('\ApplicationMeditation\\meditation_interval_chime', '2'),
+    ('\ApplicationMeditation\\meditation_preparation_time', '5'),
+    ('\ApplicationMeditation\\meditation_show_counter', '1');
```

Comment:
Please contact me for info about how to correctly update db schemas. A lot has changed since introducing UDM :)

Reply:
done